docs/index.rst

```diff
@@ -71,22 +71,19 @@ This tool helps transform legacy configuration files templated with the atparse
 
 | :any:`CLI documentation with examples<cli_template_translate_examples>`
 
-File Provisioning
-^^^^^^^^^^^^^^^^^
+File/Directory Provisioning
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^

| **CLI**: ``uw fs -h``
| **API**: ``import uwtools.api.fs``

```

Each of the other modes have this hint. Does it make sense to add it for this mode?